fakeredis.py

```diff
@@ -547,6 +547,9 @@ def type(self, name):
             return b'list'
         elif isinstance(key, set):
             return b'set'
+        else:
+            assert key is None
```

Comment:
If the key is not `None`, it means the code is missing a case and the return value of the function will be wrong anyway (it will return `None` instead of a string).